### PR TITLE
feat(UI): CNX-9070 update connectors to use new fe2 terminology

### PR DESCRIPTION
**Pull request by JR-Morgan**
- Updated naming to FE2
- Updated the "Open Stream in Web" button to go to the specific branch + commit, not just the stream (and ofcourse, working with FE1 and FE2 accounts
- Marked a couple functions as deprecated (Code that were not using, or maintaining)

### CODE REVIEW

File: bpy_speckle/operators/commit.py [hunk 1]
```diff
@@ -11,19 +11,19 @@
 
 class DeleteCommit(bpy.types.Operator):
     """
-    Deletes the selected commit from the selected stream.
+    Permanently deletes the selected version from the selected model.
```
Comment (clairekuang) [inline bpy_speckle/operators/commit.py:14]: is the change from `project` (stream) to `model` intentional?

File: bpy_speckle/operators/commit.py [hunk 2]
```diff
@@ -11,19 +11,19 @@
...
     To execute from code, call: `bpy.ops.speckle.delete_commit(are_you_sure=True)`
     """
 
     bl_idname = "speckle.delete_commit"
-    bl_label = "Delete commit"
+    bl_label = "Delete Version"
```
Comment (clairekuang) [inline bpy_speckle/operators/commit.py:19]: I believe we are using lowercase convention except where captialization is needed grammatically

File: bpy_speckle/operators/streams.py [hunk 3]
```diff
@@ -420,18 +416,40 @@ def view_stream_data_api(self, context: Context) -> None:
...
+    
 class AddStreamFromURL(bpy.types.Operator):
     """
-    Add / select a stream using its url
+    Add / select an existing project by providing its URL
     """
 
     bl_idname = "speckle.add_stream_from_url"
-    bl_label = "Add stream from URL"
+    bl_label = "Add Project From URL"
```
Comment (clairekuang) [inline bpy_speckle/operators/streams.py:448]: same with lowercase convention

**Review by clairekuang — APPROVED** [review]
Looks good to me - I see you've switched to title case for the labels, feel free to ignore earlier comments on lowercase capitalization!

File: bpy_speckle/operators/commit.py (hunk 1 above)
Comment (JR-Morgan) [reply bpy_speckle/operators/commit.py:14]: It is, I was trying to align with the new api. Where versions are a property of a model, and not of a stream.
But... perhaps we could discuss this more as a team, and align on our naming here.

**Pull request merged**